test/compilable/b1215.d

```diff
@@ -0,0 +1,118 @@
+// PERMUTE_ARGS:
+// REQUIRED_ARGS:
+
+/*
+TEST_OUTPUT:
+---
+TEST: base use case
+TEST: chained types
+TEST: chained packs
+TEST: expr
+TEST: Nested + index eval
+TEST: index with constexpr
+TEST: Nested + index with constexpr
+TEST: alias, base use case
+TEST: alias, chained types
+TEST: alias, chained packs
+TEST: alias, expr
+TEST: alias, Nested + index eval
+TEST: alias, index with constexpr
+TEST: alias, Nested + index with constexpr
+---
+*/
+
+import std.stdio;
+
+struct A(Args...)
+{
+    enum i = 1;
+
+    pragma(msg, "TEST: base use case");
```

Comment:
Sorry, I misunderstood the purpose of `pragma`s.
No one will read the printed messages so those never be printed to console during test suite running. Please replace them with code comments, and remove the unneeded `TEST_OUTPUT` section.